atomic_reactor/plugins/pre_download_remote_source.py

```diff
@@ -58,6 +58,22 @@ def get_remote_source_config(self, session, url, insecure=False):
         response.raise_for_status()
         return response.json()
 
+    def generate_cachito_env_file(self):
+        """
+        Generate cachito.env file with exported environment variables received from
+        cachito request.
+        """
+
+        if self.buildargs:
+            self.log.info('Create %s file with environment variables '
+                          'received from cachito request', CACHITO_ENV_FILENAME)
+
+            abs_path = os.path.join(REMOTE_SOURCE_DIR, CACHITO_ENV_FILENAME)
```

Comment:
The file should actually be written to `REMOTE_SOURCE`, not `REMOTE_SOURCE_DIR`. I may have caused some confusion, my bad :D

It should be the same dest dir where the configuration files go: https://github.com/containerbuildsystem/atomic-reactor/blob/9e150e3258017afd662478a0dcb4cbd9c963cc12/atomic_reactor/plugins/pre_download_remote_source.py#L78-L79

Reply:
Fixed :)